src/js/Components/LottoWinForm.js

```diff
@@ -0,0 +1,74 @@
+import Component from "../Core/Component.js";
+import { SET_WINNING_NUMBERS, SET_BONUS_NUMBER } from "../modules/actions.js";
+import { setWinningNumbers, setBonusNumber } from "../modules/creator.js";
+import { store } from "../modules/store.js";
+import { $ } from "../utils/dom.js";
+
+const actionMap = {
+  SET_WINNING_NUMBERS: (numArr) => {
+    store.dispatch(setWinningNumbers(numArr));
+  },
+
+  SET_BONUS_NUMBER: (num) => {
+    store.dispatch(setBonusNumber(num));
+  },
+};
+
+export default class LottoWinForm extends Component {
+  constructor(target) {
+    super(target);
+  }
+
+  setEvent(target) {
+    this.buttonEvent(target);
+  }
+
+  buttonEvent() {
+    $("#show-result-btn").addEventListener("click", () => {
+      const numArr = [];
+      const winningNums = document.querySelectorAll(".winning-number");
+      const bonusNum = document.querySelector(".bonus-number").value;
+
+      winningNums.forEach((node) => numArr.push(node.value));
```

Comment:
```suggestion
      winningNums.forEach((node) => numArr.push(node.value));
```
push를 사용하지않고 call메서드를 활용해서 이렇게도 작성할수있어요
const _winningNum = Array.prototype.map.call(winningNums,(node) => node.value);